src/get-started/install/_get-sdk-win_v2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Windows install
+description: How to install on Windows v2.
+short-title: Windows
+next:
+  title: Set up an editor
+  path: /get-started/windows-quick
+---
+
+## Get the Flutter SDK
+
+
+ 1. Download the following installation bundle to get the latest
+    {{site.sdk.channel}} release of the Flutter SDK:
+
+    [(loading...)](#){:.download-latest-link-{{os}}.btn.btn-primary}
+
+    For other release channels, and older builds,
+    see the [SDK releases][] page.
+ 1. Extract the zip file and place the contained `flutter`
+    in the desired installation location for the Flutter SDK
+    (for example, `C:\src\flutter`).
+
+{{site.alert.warning}}
+  Do not install Flutter to a path that contains special
+  characters or spaces.
+{{site.alert.end}}
+
+{{site.alert.warning}}
+  Do not install Flutter in a directory like 
+  `C:\Program Files\` that requires elevated privileges.
+{{site.alert.end}}
+
+If you don't want to install a fixed version of the installation 
+bundle, you can skip steps 1 and 2. Instead, get the source code 
+from the [Flutter repo][] on 
+GitHub, and change branches or tags as needed. For example:
+
+```batchfile
+C:\src>git clone https://github.com/flutter/flutter.git -b stable
+```
+
+You are now ready to run Flutter commands in the Flutter Console.
+
+[Flutter repo]: {{site.repo.flutter}}
+
+### Update your path
+
+If you wish to run Flutter commands in the regular Windows console,
+take these steps to add Flutter to the `PATH` environment variable:
+
+* From the Start search bar, enter 'env'
+  and select **Edit environment variables for your account**.
+* Under **User variables** check if there is an entry called **Path**:
+  * If the entry exists, append the full path to `flutter\bin` using
+    `;` as a separator from existing values.
+  * If the entry doesn't exist,
+    create a new user variable named `Path` with
+    the full path to `flutter\bin` as its value.
+
+You have to close and reopen any existing console windows
+for these changes to take effect.
```

Comment:
Nit: you don't necessarily need to close the terminal for these changes to take effect, consider simplifying to:

```suggestion
You will need to launch a new console window
for these changes to take effect.
```